modules/multiplayer/tests/test_scene_multiplayer.h

```diff
@@ -215,7 +204,7 @@ TEST_CASE("[Multiplayer][SceneMultiplayer][SceneTree] Send Authentication") {
 
 		CHECK_EQ(scene_multiplayer->poll(), Error::OK);
 
-		SIGNAL_CHECK("peer_authentication_failed", build_array(build_array(first_peer_id), build_array(second_peer_id)));
+		SIGNAL_CHECK("peer_authentication_failed", Array({ { first_peer_id }, { second_peer_id } }));
```

Comment:
This and a couple of the ones in `test_text_edit` need to explicitly be an array, otherwise the macro seems to fail for some reason.

Reply:
Maybe it would be more readable like this?

```cpp
Array result = { { first_peer_id }, { second_peer_id } };
SIGNAL_CHECK("peer_authentication_failed", result);
```

Reply:
I don't feel it's that much more readable, and I would rather not need to add a local variable every time `SIGNAL_CHECK` is called.
`SIGNAL_CHECK` also always only takes in an array of expected signal args, so it should always be clear what it is.